script/preCache.js

```diff
@@ -9,17 +9,21 @@ module.exports = breadboard({
     'package.json': require('../package.json')
   },
   entry: ({
+    '/config/index': getConfig,
     '/lib/bundler/index': bundle,
     '/lib/getComponentInfo': getComponentInfo
   }) => {
+    const config = getConfig();
     const components = getComponentInfo().map(componentInfo => componentInfo.name );
+    const preCacheComponentBundles = config.components && config.components.preCacheBundles ? config.components.preCacheBundles : [];
```

Comment:
That's completely clear now that I read it again. I'm not sure what I was thinking yesterday.